package.json

```diff
@@ -207,6 +207,23 @@
           "type": "boolean",
           "default": true
         },
+        "codecov.fileDecorations.show": {
+          "description": "Whether to decorate each file in the file tree with a meter representing its coverage",
+          "type": "boolean",
+          "default": false
+        },
+        "codecov.fileDecorations.low": {
+          "type": "number",
+          "default": 70
+        },
+        "codecov.fileDecorations.high": {
+          "type": "number",
+          "default": 85
+        },
+        "codecov.fileDecorations.optimum": {
+          "type": "number",
+          "default": 100
```

Comment:
Is this `default` field used actually populate the settings value? If so, can we avoid setting defaults in `resolveSettings` (or de-duplicate the default values by referencing these values in the manifest?)